root/artist/edit_form.tt

```diff
@@ -97,16 +101,93 @@
   </div>
 
   <div class="documentation">
-    [%- area_bubble() -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the artist’s official name.',
+             { doc => { href => doc_link('Artist#Name'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
 
     [%- sort_name_bubble() -%]
 
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field is used to help users distinguish between identically‐ or similarly‐named artists.
+              It’s okay to leave it blank if it seems unlikely that there will be other artists with similar names.',
+             { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('This field is not a place to store general background information.
+              That kind of information should go in an {doc|annotation}.',
+             { doc => { href => doc_link('Annotation'), target => '_blank' } }) %]
+      </p>
+    </div>
+
     [%- type_bubble(form.field('type_id'), artist_types) -%]
 
+    <div class="bubble" id="gender-bubble">
+      <p>
+        [% l('The {doc|gender} field describes the gender with which a person or character identifies.',
+             { doc => { href => doc_link('Artist#Gender'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Gender'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="area-bubble">
+      <p>
+        [% l('The {doc|area} indicates the area the artist is primarily identified with.',
+             { doc => { href => doc_link('Artist#Area'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Main_area'), target => '_blank' } }) %]
+      </p>
+      [% area_bubble_selection() %]
+    </div>
+
     [%- ipi_bubble() -%]
     [%- isni_bubble() -%]
-  </div>
 
+    <div class="bubble" id="begin-end-date-bubble">
+      <p>
+        [% l('The {doc|begin and end dates} describe when the artist started and stopped existing.',
```

Comment:
While "started and stopped existing" makes sense as a generic wording for all sorts of artists, it sounds super weird for talking about people - it's a bit like those youtubers saying "unalive" :sweat_smile: 

I wonder if it would make more sense to have per-type strings here.

Reply:
Yeah, I didn't love this either. I based it on the docs, which somehow feel even more awkward to me:

> The begin and end dates indicate when an artist started and finished its existence.

I'll try to update this and the begin/end area text to display appropriate messages for different types.

Reply:
What's the preferred way to do this in this template? I don't think that Knockout is being used in `root/*/edit_form.tt`. Should I just add some JS to `root/static/scripts/artist/edit.js` that's similar to what's being done in `root/static/scripts/edit/typeBubble.js`?

Reply:
@reosarevok and @mwiencek, any thoughts about the best way to do this?